components/post_view/post_time.jsx

```diff
@@ -47,18 +47,24 @@ export default class PostTime extends React.PureComponent {
 
     renderTimeTag() {
         const date = new Date(this.props.eventTime);
+        const militaryTime = this.props.useMilitaryTime;
 
-        const hour = this.props.useMilitaryTime ? date.getHours() : (date.getHours() % 12 || 12);
+        const hour = militaryTime ? date.getHours() : (date.getHours() % 12 || 12);
         let minute = date.getMinutes();
         minute = minute >= 10 ? minute : ('0' + minute);
+        let time = '';
+
+        if (!militaryTime) {
+            time = (hour >= 12 ? ' PM' : ' AM');
```

Comment:
For civilian time, `hour` is always 1-12 at this point. 'AM' will be displayed for 22 hours a day.